### PR TITLE
feat(cce/cluster): support updating eip in cce cluster

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

support updating eip in cce cluster

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
support updating eip in cce cluster
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccCCEClusterV3_withEip'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccCCEClusterV3_withEip -timeout 360m -parallel 4
=== RUN   TestAccCCEClusterV3_withEip
=== PAUSE TestAccCCEClusterV3_withEip
=== CONT  TestAccCCEClusterV3_withEip
--- PASS: TestAccCCEClusterV3_withEip (577.73s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       577.880s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_cce_cluster_v3_test.go [hunk 1]
```diff
@@ -358,6 +365,47 @@ resource "huaweicloud_cce_cluster" "test" {
...
+	publicip {
+	  type = "5_bgp"
+	}
+	bandwidth {
+	  name        = "test"
+	  size        = 8
+	  share_type  = "PER"
+	  charge_mode = "traffic"
+	}
+  }
```
Comment (Lance52259) [inline huaweicloud/resource_huaweicloud_cce_cluster_v3_test.go:387]: wrong format

Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_cluster_v3_test.go:387]: fixed

File: huaweicloud/resource_huaweicloud_cce_cluster_v3.go [hunk 2]
```diff
@@ -823,3 +843,24 @@ func resourceCCEClusterV3Awake(ctx context.Context, d *schema.ResourceData, cceC
 	}
 	return nil
 }
+
+func resourceCCEClusterV3EipAction(ctx context.Context, cceClient, eipClient *golangsdk.ServiceClient,
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_cce_cluster_v3.go]: it seems that `ctx` is not used

Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_cluster_v3.go]: sure, fixed

**Pull request merged**